testsuite/OMSimulator/import_export_signalFilter.lua

```diff
@@ -0,0 +1,201 @@
+-- 						</ssd:Connectors>
+-- 						<ssd:ParameterBindings>
+-- 							<ssd:ParameterBinding>
+-- 								<ssd:ParameterValues>
+-- 									<ssv:ParameterSet version="1.0" name="parameters">
+-- 										<ssv:Parameters>
+-- 											<ssv:Parameter name="parameter_1">
+-- 												<ssv:Real value="20" />
+-- 											</ssv:Parameter>
+-- 											<ssv:Parameter name="Input_1">
+-- 												<ssv:Real value="10" />
+-- 											</ssv:Parameter>
+-- 										</ssv:Parameters>
+-- 									</ssv:ParameterSet>
+-- 								</ssd:ParameterValues>
+-- 							</ssd:ParameterBinding>
+-- 						</ssd:ParameterBindings>
+-- 						<ssd:Annotations>
+-- 							<ssc:Annotation type="org.openmodelica">
+-- 								<oms:Annotations>
+-- 									<oms:SimulationInformation>
+-- 										<oms:VariableStepSolver description="cvode" absoluteTolerance="0.000100" relativeTolerance="0.000100" minimumStepSize="0.000100" maximumStepSize="0.100000" initialStepSize="0.000100" />
+-- 									</oms:SimulationInformation>
+-- 								</oms:Annotations>
+-- 							</ssc:Annotation>
+-- 						</ssd:Annotations>
+-- 					</ssd:System>
+-- 					<ssd:Component name="Gain" type="application/x-fmu-sharedlibrary" source="resources/0001_Gain.fmu">
+-- 						<ssd:Connectors>
+-- 							<ssd:Connector name="u" kind="input">
+-- 								<ssc:Real />
+-- 								<ssd:ConnectorGeometry x="0.000000" y="0.500000" />
+-- 							</ssd:Connector>
+-- 							<ssd:Connector name="y" kind="output">
+-- 								<ssc:Real />
+-- 								<ssd:ConnectorGeometry x="1.000000" y="0.500000" />
+-- 							</ssd:Connector>
+-- 							<ssd:Connector name="k" kind="parameter">
+-- 								<ssc:Real />
+-- 							</ssd:Connector>
+-- 						</ssd:Connectors>
+-- 					</ssd:Component>
+-- 				</ssd:Elements>
+-- 				<ssd:Annotations>
+-- 					<ssc:Annotation type="org.openmodelica">
+-- 						<oms:Annotations>
+-- 							<oms:SimulationInformation>
+-- 								<oms:FixedStepMaster description="oms-ma" stepSize="0.100000" absoluteTolerance="0.000100" relativeTolerance="0.000100" />
+-- 							</oms:SimulationInformation>
+-- 						</oms:Annotations>
+-- 					</ssc:Annotation>
+-- 				</ssd:Annotations>
+-- 			</ssd:System>
+-- 			<ssd:DefaultExperiment startTime="0.000000" stopTime="1.000000">
+-- 				<ssd:Annotations>
+-- 					<ssc:Annotation type="org.openmodelica">
+-- 						<oms:Annotations>
+-- 							<oms:SimulationInformation resultFile="model_res.mat" loggingInterval="0.000000" bufferSize="10" signalFilter="resources/signalFilter.xml" />
+-- 						</oms:Annotations>
+-- 					</ssc:Annotation>
+-- 				</ssd:Annotations>
+-- 			</ssd:DefaultExperiment>
+-- 		</ssd:SystemStructureDescription>
+-- 	</oms:ssd_file>
+-- 	<oms:signalFilter_file name="resources/signalFilter.xml">
+-- 		<oms:SignalFilter version="1.0">
+-- 			<oms:Variable name="model.root.System2.Input_1" />
+-- 			<oms:Variable name="model.root.Gain.u" />
+-- 			<oms:Variable name="model.root.Gain.k" />
```

Comment:
Would it make sense to include the signal type as well?

Reply:
this is fixed with https://github.com/OpenModelica/OMSimulator/pull/915/commits/0b50246e2a14b3ddeef743c41a03cfbcae5075d9